packages/position-managers/src/constants/vaults/index.ts

```diff
@@ -16,5 +16,5 @@ export const VAULTS_CONFIG_BY_CHAIN = {
 }
 
 export function isPCSVaultConfig(config: VaultConfig): config is PCSDuoTokenVaultConfig {
-  return config.manager === MANAGER.PCS || config.manager === MANAGER.BRIL
+  return config.manager === MANAGER.PCS || config.manager === MANAGER.BRIL || config.manager === MANAGER.RANGE
```

Comment:
hmm I thought this function is used to tell if the vault is PCS vault or third party vaults?

Reply:
ok, let me make some adjustment